src/vs/workbench/services/extensions/worker/webWorkerExtensionHostIframe.html

```diff
@@ -73,6 +74,10 @@
 			}
 
 			const worker = new Worker(workerUrl, { name });
+			if (locale) {
+				// send before `vs/workbench/api/worker/extensionHostWorker`load event.
+				worker.postMessage(`worker:init-locale:${locale}`);
```

Comment:
Because the worker cannot read the localstorge directly and the iframe may not be in the same domain name as the main application, the method used is to obtain the language in the host, pass it to the iframe, and then pass it to the worker.

I don't know if this will be strange